### PR TITLE
feat: adding Scarf pixels to gather telemetry on readme and website

### PR DESCRIPTION
**Pull request by rusackas**
### SUMMARY

This PR adds two distinct tracking pixels for telemetry, using the Scarf platform. One is added to the readme to capture analytics around visits to the repo, and one captures visits to the website/docs.

All of this data will be available to interested PMC members, and reported periodically in things like Superset Board Reports and Superset Town Hall events.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

N/A

### TESTING INSTRUCTIONS

Log into Scarf and see the charts ticking!

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -252,7 +252,9 @@ const config = {
               <a href="https://apache.org/events/current-event" target="_blank" rel="noreferrer">Events</a>&nbsp;|&nbsp;
               <a href="https://apache.org/licenses/" target="_blank" rel="noreferrer">License</a>
             </small>
-          </p>`,
+          </p>
+          <img referrerPolicy="no-referrer-when-downgrade" src="https://static.scarf.sh/a.png?x-pxid=39ae6855-95fc-4566-86e5-360d542b0a68" />
```
Comment (villebro) [inline docs/docusaurus.config.js:256]: Should we add `<!-- telemetry/analytics pixel: -->` here, too?

**Review by villebro — COMMENTED** [review]
Minor comment

File: docs/docusaurus.config.js [hunk 2]
```diff
@@ -252,7 +252,9 @@ const config = {
               <a href="https://apache.org/events/current-event" target="_blank" rel="noreferrer">Events</a>&nbsp;|&nbsp;
               <a href="https://apache.org/licenses/" target="_blank" rel="noreferrer">License</a>
             </small>
-          </p>`,
+          </p>
```
Comment (rusackas) [inline docs/docusaurus.config.js:255]:
```suggestion
          </p>
          <!-- telemetry/analytics pixel: -->
```

**Review by villebro — APPROVED** [review]
LGTM, looking forward to seeing the stats, this will really help improve our docs

**Comment by villebro** [comment]
@rusackas btw, there's also `CONTRIBUTING.md` that we may want to track, and maybe the `resources/INTHEWILD.md` and other friends, too

**Comment by rusackas** [comment]
I'll add something to the docs and/or readme about telemetry about the aggregated data collected from processing (and the deleting) your IP, though ironically you would have been pixellated(?) by the time you've read it.

**Comment by rusackas** [comment]
> @rusackas btw, there's also `CONTRIBUTING.md` that we may want to track, and maybe the `resources/INTHEWILD.md` and other friends, too

Yes indeed... and the wiki, potentially... lots more to expand into.

**Review by eschutho — APPROVED** [review]

**Pull request merged**